AxonDeepSeg/morphometrics/launch_morphometrics_computation.py

```diff
@@ -146,7 +146,9 @@ def main(argv=None):
                                         ('y0', 'f4'),
                                         ('gratio', 'f4'),
                                         ('axon_area', 'f4'),
+                                        ('axon_perimeter', 'f4'),
                                         ('myelin_area', 'f4'),
+                                        ('axonmyelin_perimeter', 'f4'),
```

Comment:
We should have the stats in the same order as in the fsleyes plugin.

Reply:
I second on that. Fixed in the latest commit.